package.json

```diff
@@ -44,13 +46,18 @@
   "engines": {
     "node": ">=18.0.0"
   },
+  "exports": {
```

Comment:
Doesn't this prevent imports of specific sub-files? I think that's okay since we export most stuff from the main package but wanted to make sure that we are not preventing extensibility.

Reply:
Is there an official way to test it?

Reply:
I'm almost sure we would only disallow importing from other files if the exports field was defined as follows:

```
"exports": {
  ".": {
    "import": "./dist/index.js",
    "require": "./cjs/index.js"
  }
},
```